spec/problem2_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'problem2'
+describe 'problem2' do
+  it 'returns the two largest values from array' do
+    expect(problem2([1,2,3,4,5])).to match_array [4,5]
```

Comment:
It is generally frowned upon to have multiple assertions in a test, why do you think that is?

Reply:
I wanted to say that from a practical point of view that the failure can be pinpointed from the test "summary" instead of having to review the Failures ("verbose") section of the test run. However can't take credit for that claim any more cause I've opened up the link @pirvudoru shared and it's recommended there with similar justification :)